ansible/roles/debops.rails_deploy/tasks/deploy.yml

```diff
@@ -139,20 +150,25 @@
            enabled={{ item.enabled | default(True) }}
   when: rails_deploy_git_location is defined and rails_deploy_git_location and
         rails_deploy_extra_services
-  with_items: rails_deploy_extra_services
+  with_items: '{{ rails_deploy_extra_services }}'
 
 - name: Execute shell commands after the backend is ready
   shell: '{{ rails_deploy_env_source }} && {{ item }} chdir={{ rails_deploy_src }}'
   when: rails_deploy_post_restart_shell_commands and rails_deploy_git_location and
         rails_deploy_register_repo_status is defined and
         rails_deploy_register_repo_status is changed
-  with_items: rails_deploy_post_restart_shell_commands
+  with_items: '{{ rails_deploy_post_restart_shell_commands }}'
+  become: True
   become_user: '{{ rails_deploy_service }}'
 
 - name: Disable the temporary deploy page
-  command: rm -f public/index.html chdir={{ rails_deploy_src }}
+  command: rm -f public/index.html
+  args:
+    chdir: '{{ rails_deploy_src }}'
+    warn: False
```

Comment:
This can also be expressed as file: state=absent. Then you also don’t need the `warn: False` ;-)

Reply:
I know, but to fix `ansible-lint` issues in this PR I wanted to do as few changes to the existing code as possible, since it's a large set of patches. Tasks like this can be converted later to proper Ansible modules.

Reply:
Than maybe add a FIXME note to the files. My reasoning is that `ansible-lint` makes you aware of things that can be improved. When you do as little changes as possible if just feels like you hack the linter in ways that it will stop complying for example by disabling warnings while there is a proper way to do it.